inc/audiences/namespace.php

```diff
@@ -8,6 +8,7 @@
 namespace Altis\Analytics\Audiences;
 
 use Altis\Analytics\Utils;
+use function Altis\Analytics\Utils\time_ago_in_milliseconds;
```

Comment:
It's not enforced in our coding standards yet but we're discouraging the use of `use function` in favour of using the namespace and referencing the function as `Namespace\function()` so it's clearer where the function comes from in situ.

Reply:
```suggestion
```